test/unit/resources/aws_iam_password_policy_test.rb

```diff
@@ -75,7 +75,7 @@ def test_policy_to_s
 
   private 
 
-  def configure_policy_password_reuse_prevention(value: value, n: 1)
+  def configure_policy_password_reuse_prevention(value:, n: 1)
```

Comment:
This seems weird. Is this canonical Ruby to have a key param without a value? Maybe we should be passing `value` as a required param.